README.md

```diff
@@ -1,46 +1,37 @@
-# Getting Started with Create React App
+<h1 align="center">Space Burger</h1>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">Web application for ordering in a burger shop using the burger constructor</div>
 
-## Available Scripts
+![Tech logos](https://i.ibb.co/DVFj8PL/tech-icons.jpg)
 
-In the project directory, you can run:
+![App screenshot](./readme_src/screenshot-app.PNG)
 
-### `yarn start`
+## What is this web application for? 🤷‍♀️
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+It is an interactive application that allows the user to conveniently assemble their own special burger using the constructor.
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+Also for registered users, you can view the order history. The user can see the order feed - a list of burgers that are being prepared right now or are already ready. This functionality is available live thanks to the web socket connection.
 
-### `yarn test`
+## Features
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+- Registration and authorization of users
```

Comment:
- Registration and authorization user.
- The ability to construct your special burger using drag and drop.
- View user's order history.
- Overview already assembled and prepared burgers.
- Ability to edit your account data.